gammapy/utils/table.py

```diff
@@ -86,6 +93,9 @@ def table_row_to_dict(row, make_quantity=True):
     for name, col in row.columns.items():
         val = row[name]
 
+        if isinstance(val, np.ma.core.MaskedConstant):
+            val = GammapyMaskedConstant()
```

Comment:
Isn't it simpler to write:
`val = str(MaskedConstant())`
instead of using the overloaded class?

Reply:
done (required also a little change in the hgps catalog object to work)